### PR TITLE
[docs] Documentation Updates

### PR DESCRIPTION
**Pull request by KCarretto**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:

Updates our docs and makes them more organized.

### CODE REVIEW

**Review by jabbate19 — APPROVED** [review]
LGTM

File: docs/_docs/admin-guide/tavern.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+```
+
+```json
+{
+  "input": {
+    "name": "Run test tome",
+    "tomeID": "21474836488",
+    "parameters": "{\"print_string\":\"Hello World\"}"
+  },
+  "sess": ["8589934593"]
```
Comment (hulto) [inline docs/_docs/admin-guide/tavern.md:357]: I think this needs to be updated to `"beaconIDs"`
image

File: docs/_docs/admin-guide/tavern.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+```json
+{
+  "input": {
+    "name": "Test tome",
+    "description": "Just a sample",
+    "eldritch": "print(input_params['print_string'])",
+    "fileIDs": [],
+    "paramDefs": "[{\"name\":\"print_string\",\"label\":\"Print String\",\"type\":\"string\",\"placeholder\":\"A message to print\"}]"
+  }
+}
```
Comment (hulto) [inline docs/_docs/admin-guide/tavern.md:337]: Gotta update to:
```json
{
  "input": {
    "name": "Test tome",
    "author":"Hulto",
    "description": "Just a sample",
    "paramDefs": "[{\"name\":\"print_string\",\"label\":\"Print String\",\"type\":\"string\",\"placeholder\":\"A message to print\"}]",
    "eldritch": "print(input_params['print_string'])",
    "fileIDs": [],
    "tactic": "IMPACT"
  }
}
```
Where `tactic` is a MITRE ATT&CK technique.

File: docs/_docs/admin-guide/tavern.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+  beacons {
+    id
+    identifier
+    name
+    hostPlatform
+    hostIdentifier
+    hostPrimaryIP
+  }
+}
+```
```
Comment (hulto) [inline docs/_docs/admin-guide/tavern.md:315]:
```graphql
query get_beacons {
    beacons {
        id
        identifier
        name
        host {
          platform
          identifier
          primaryIP
        }
    }
}
```

**Review by hulto — CHANGES_REQUESTED** [review]
Saw new changes refreshing

**Review by hulto — APPROVED** [review]

**Pull request merged**